components/VerticalTabs/Pager.tsx

```diff
@@ -6,27 +8,51 @@ interface PagerProps {
 }
 
 const Pager = ({ identifier, count, current, hidden }: PagerProps) => {
+    const [isDesktop, setDesktop] = useState(false);
+
     const scrollTo = (idx: number) => {
         document
             .getElementById(`carousel-${identifier}-${idx}`)
             ?.scrollIntoView({ block: "center" });
     };
 
+    useEffect(() => {
+        if (window.innerWidth > 1450) {
+          setDesktop(true);
+        } else {
+          setDesktop(false);
+        }
+    
+        const updateMedia = () => {
+          if (window.innerWidth > 1450) {
+            setDesktop(true);
+          } else {
+            setDesktop(false);
+          }
+        };
+        window.addEventListener('resize', updateMedia);
+        return () => window.removeEventListener('resize', updateMedia);
+      }, []);
+    
+
```

Comment:
We can accompllish this using [media queries](https://tailwindcss.com/docs/responsive-design)

Instead of this useEffect/useState pattern... (see below comment)